pkg/diagnostics/v2/client.go

```diff
@@ -24,6 +24,11 @@ type Bundle struct {
 	Errors  []string  `json:"errors,omitempty"`
 }
 
+// IsFinished returns true if the bundle has a status that would indicate that it is finished being created.
```

Comment:
`IsFinished returns if the bundle has a status that indicating that it is finished.`